astropy/units/format/unicode_format.py

```diff
@@ -5,7 +5,9 @@
 """
 
 
-from . import console
+import astropy.units as u
```

Comment:
I think this is preferred?
```suggestion
from astropy import units as u
```

Reply:
I don't think we need the import (see other comment) but we seem to have different "defaults" for this. I seem to have settled on using `from ... import X` for cases where `X` is not a module...